fractions/microprofile/microprofile-fault-tolerance/src/main/java/org/wildfly/swarm/microprofile/faulttolerance/MicroProfileFaultToleranceFraction.java

```diff
@@ -16,27 +16,24 @@
 
 package org.wildfly.swarm.microprofile.faulttolerance;
 
-import org.wildfly.swarm.config.runtime.AttributeDocumentation;
-import org.wildfly.swarm.spi.api.Defaultable;
+import io.smallrye.faulttolerance.HystrixCommandInterceptor;
 import org.wildfly.swarm.spi.api.Fraction;
-import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.Module;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule.MetaInfDisposition;
 
 /**
  * @author Antoine Sabot-Durand
  */
-@DeploymentModule(name = "org.wildfly.swarm.microprofile.faulttolerance", export = true, slot = "deployment", metaInf = MetaInfDisposition.IMPORT)
+@DeploymentModule(name = "io.smallrye.faulttolerance", metaInf = MetaInfDisposition.IMPORT, export = true, services = Module.ServiceHandling.IMPORT)
 public class MicroProfileFaultToleranceFraction implements Fraction<MicroProfileFaultToleranceFraction> {
 
     public MicroProfileFaultToleranceFraction() {
-    }
+        String synchronousCircuitBreaker = System.getProperty("swarm.microprofile.fault-tolerance.synchronous-circuit-breaker");
```

Comment:
I'm not so sure this would work. `@Configurable` is repeatable, so you can try to add another `@Configurable()` alias to the config property instead. 

By the way, I've created https://github.com/smallrye/smallrye-fault-tolerance/issues/10 ...

Anyway, a simple test would be good.